ocs_ci/deployment/aws.py

```diff
@@ -119,6 +135,10 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
             log_cli_level (str): openshift installer's log level
                 (default: "DEBUG")
         """
+        if self.check_cluster_existence() and not (
```

Comment:
I would add first this one `not config.DEPLOYMENT.get('force_deploy_multiple_clusters')` as if this one will be set it will not run the function self.check_cluster_existence()  at all I think so will safe few seconds.

Reply:
done